graders/r/README.md

```diff
@@ -1,10 +1,31 @@
-## PrairieLearn Autograding R Docker Image
+## rocker-pl
 
+This is a Docker image working as an external grader to be paired with PrairieLearn.  It has been
+used in the [STAT 430](https://stat430.com) (2018-2020) and [STAT 447](https://stat430.com) (since
+2021) courses on _Data Science Programming Methods_ at the University of Illinois at
+Urbana-Champaign. In a previous course iteration, a CentOS7-based container was used via Python. We
+now drive R directly from PrairieLearn evaluating code in this container. However, the container is
+perfectly generic and can be used for general R work as well.
 
-The docker container is from the [versioned `tidyverse` stack](https://github.com/rocker-org/rocker-versioned)
-([3.6.1 source](https://github.com/rocker-org/rocker-versioned/blob/master/tidyverse/Dockerfile))
-created by the [`rocker`](https://www.rocker-project.org/) project. Ontop of the 
-versioned `tidyverse` container, we've included packages additional _R_ packages
-and a specific user, called `ag`, for PrairieLearn's autograde routine.
+It is based on the [Rocker](https://rocker-project.org) container `r-ubuntu` in order to take
+advantage of the prebuilt Ubuntu binaries available for the stable 'long-term support' (LTS)
+releases---both current versions of R itself, as well as current CRAN packages.
 
+### R Packages Installed
 
+See the Dockerfile for full details but we include
+
+- tidyverse and data.table for data manipulation;
+- RUnit, testthat and tinytest for testing;
+- microbenchmark, rbenchmark and bench for timing;
+- shiny, flexdashboard, dygraphs for shiny;
+
+and more. We also include a few command-line applications such as `git` and `sqlite3`.
+
+### Authors
+
+Alton Barbehenn and Dirk Eddelbuettel
+
+### License
+
+AGPL-3
```

Comment:
No need for licenses on individual source components.
```suggestion
```

Reply:
Done in bc30a9661 (after a rebase too).